TeeWhyKay/Chomp
# Feature/email results of chomp session

## Why

To have email to our users

## What

This pull request introduces the following:

 * Email to user telling them their account is created
 * Email to user telling them their Chomp session is created

## How to Test

- Create an account, and check the email of the account
- Create a chomp session, and check the email again for another email

## Others

Try to go to my repo and see if `rails s` works.
